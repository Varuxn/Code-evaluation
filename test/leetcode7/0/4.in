8
401 233 34 681 361 332 178 178
