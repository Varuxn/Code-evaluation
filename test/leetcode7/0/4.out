168 199 34 320 29 154 0 178 